### PR TITLE
Refactor / Move icons to theme folder

### PR DESCRIPTION
**Pull request by MelissaDTH**
## Icons refactor
[OTT-188](https://videodock.atlassian.net/browse/OTT-188)

- This PR moves the icons to the `/theme` directory, to  not make them React dependant and usable throughout the project 
  - It refactors the icons from a React component to solely a `.svg` and with the help of the wrapper function `Icon` and `vite-plugin-svgr` we can obtain that `.svg` file as a React component and implement it in the app
  - Using this plugin is fine, because react-native already will need other components and UI, so by that time we can create a different Icon component for that purpose (to avoid using vite because vite will not available there)
  - Sorry for the convoluted PR due to the updated snapshots and such, couldn't keep it smaller!

[OTT-188]: https://videodock.atlassian.net/browse/OTT-188?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ui-react/src/components/Icon/Icon.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import styles from './Icon.module.scss';
+
+export type Props = {
+  icon: FC<SVGProps<SVGSVGElement>>;
+  className?: string;
+};
+
+const Icon: FC<Props> = ({ icon: IconComponent, className }) => {
+  return <IconComponent className={classNames(styles.icon, className)} />;
```
Comment (langemike) [inline packages/ui-react/src/components/Icon/Icon.tsx]: We need the `aria-hidden="true"` back 😄 I think we do not need the focusable="false" as reading some articles on internet. `aria-hidden="true"` is sufficient

```suggestion
  return <IconComponent aria-hidden="true" className={classNames(styles.icon, className)} />;
```

**Review by langemike — COMMENTED** [review]
What a burden! Glad to see you were able to fix it. I agree with the directory approach you took. Only 1 (major) issue, we need the `aria-hidden="true"` back on the icon!

File: packages/theme/assets/icons/arrow_left.svg
Comment (ChristiaanScheermeijer) [inline packages/theme/assets/icons/arrow_left.svg:1]: Is this file used? It seems to be the wrong icon 😅

Comment (MelissaDTH) [reply packages/theme/assets/icons/arrow_left.svg:1]: Haha yes I **just** fixed this 1 minute ago!

File: packages/ui-react/src/components/Icon/Icon.tsx (hunk 1 above)
Comment (MelissaDTH) [reply packages/ui-react/src/components/Icon/Icon.tsx]: Thanks Mike!

**Review by langemike — APPROVED** [review]

File: packages/ui-react/src/components/PasswordField/PasswordField.tsx [hunk 2]
```diff
@@ -40,7 +41,7 @@ const PasswordField: React.FC<Props> = ({ value, showToggleView = true, showHelp
       rightControl={
         showToggleView ? (
           <IconButton aria-label={viewPassword ? t('reset.hide_password') : t('reset.view_password')} onClick={() => toggleViewPassword()}>
-            {viewPassword ? <Visibility /> : <VisibilityOff />}
+            {viewPassword ? <Icon icon={Visibility} /> : <Icon icon={VisibilityOff} />}
```
Comment (royschut) [inline packages/ui-react/src/components/PasswordField/PasswordField.tsx]: :-) 
```suggestion
            <Icon icon={viewPassword ? Visibility : VisibilityOff} />
```

**Review by royschut — APPROVED** [review]
Nicey! 
Approved, provided that you'll remove `arrow_left_right.svg`. :)

**Pull request merged**